src/core/cache.js

```diff
@@ -86,16 +87,13 @@ export const getFromCache = (
       isConstant,
       isComputed
     );
-  } else if (
-    process.env.NODE_ENV !== "production" &&
-    namespace &&
-    !globalCache[namespace]
-  ) {
-    console.warn(
-      `useStyles Non-Existent-Namespace: Namespace "${namespace}" does not exist or has not been imported. You are seeing this warning because you are in development mode. In a production build there will be no warning and these styles will be ignored.`
+  } else if (namespace && !globalCache[namespace]){
```

Comment:
I think that if we maintain the NODE_ENV condition, the condition will also be removed from the production build.
```suggestion
  } else if (process.env.NODE_ENV !== "production" && namespace && !globalCache[namespace]){
```